doc/icache.rst

```diff
@@ -115,7 +133,7 @@ The fill buffers perform several functions in the I$ and constitute most of it's
 - The fill buffers are used as data storage for hitting requests as well as for miss tracking so all lookup requests require a fill buffer.
 - A fill buffer makes multiple external requests to memory to fetch the required data to fill a cache line (tracked via ``fill_ext_cnt_q``).
 - Returning data is tracked via ``fill_rvd_cnt_q``.
-Not all requests will fetch all their data, since requests can be cancelled due to a cache hit or an intervening branch.
+  Not all requests will fetch all their data, since requests can be cancelled due to a cache hit or an intervening branch.
```

Comment:
It looks like I was trying to make bullet points here, but failed. I guess either leave it as it is and I'll fix it later, or switch the `-`'s to `*`'s.